lib/internal/fs/sync.js

```diff
@@ -49,7 +47,7 @@ function copyFile(src, dest, mode) {
   binding.copyFileSync(
     pathModule.toNamespacedPath(src),
     pathModule.toNamespacedPath(dest),
-    getValidMode(mode, 'copyFile'),
```

Comment:
Can you rebase?

Reply:
I merged to keep the change history but I can rebase if you prefer.